src/Chapter06/Listing06.26.DefiningAPrimaryConstructor.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_26;
+
+#if NET8_0_OR_GREATER
+public class Program
+{
+    public static void Main()
+    {
+        Employee employee;
+        employee = new("Inigo", "Montoya");
+        employee.Salary = "Too Little";
+
+        System.Console.WriteLine(
+            "{0} {1}: {2}",
+            employee.FirstName,
+            employee.LastName,
+            employee.Salary);
+    }
+}
+
+#region INCLUDE
+    #region HIGHLIGHT
+    // Employee constructor
+public class Employee(string firstName, string lastName)
+{
+    public string FirstName { get; set; } = firstName;
+    public string LastName { get; set; } = lastName;
+    #endregion HIGHLIGHT
+    public string? Salary { get; set; } = "Not Enough";
+
+    #region EXCLUDE
+    public string? Title { get; set; }
+    public Employee? Manager { get; set; }
+
+    // Name property
+    public string Name
+    {
+        get
+        {
+            return FirstName + " " + LastName;
+        }
+        set
+        {
+            // Split the assigned value into 
+            // first and last names
+            string[] names;
+            names = value.Split(new char[] { ' ' });
+            if(names.Length == 2)
+            {
+                FirstName = names[0];
+                LastName = names[1];
+            }
+            else
+            {
+                // Throw an exception if the full 
+                // name was not assigned
+                throw new System.ArgumentException(
+                    string.Format(
+                    $"Assigned value '{ value }' is invalid", 
+                    nameof(value)));
```

Comment:
This is a confusing use of string.Format.
```suggestion
                throw new System.ArgumentException(
                   $"Assigned value '{ value }' is invalid", 
                    nameof(value));
```